kagenti/ui/Home.py

```diff
@@ -94,5 +97,51 @@
         """
     )
 
+# --- Login Redirect ---
+st.session_state[ENABLE_AUTH_STRING] = False
+ENABLE_AUTH = os.environ.get('ENABLE_AUTH')
+if ENABLE_AUTH == "true":
```

Comment:
Make this check case-insensitive, and consider logging the enable auth setting at debug or info level.